generate-analytical-dataset.py

```diff
@@ -10,48 +10,79 @@
 from pyspark.sql.types import *
 from pyspark.sql import Row
 
+from pysecret import AWSSecret
+
+aws_profile = "analytical_dataset_generator"
+aws_region = "eu-west-2"
+aws = AWSSecret(profile_name=aws_profile, region_name=aws_region)
```

Comment:
We won't need profiles or regions as this'll be running in an EMR instance so that will be automatically known. I also don't like that variable name, again poor docs.

`secret = AWSSecret()` I think will do the job here.

Reply:
Done.  I think my recent changes have bloated this somewhat?